docs/csharp/programming-guide/nullable-types/handling-null-and-operators.md

```diff
@@ -0,0 +1,159 @@
+# Null coalescing and operators
+
+## Introduction
+Since version 6.0 C# supports the null coalescing and null operator. With these shorthands it is easier to write null checks and thus make your code more readable. On this page is shown how to use these shorthand notations.
+
+### Null coalescing (??)
+The null coalescing operator consists of a double question mark "??". With this operator, the null check of a reference variable can be simplified greatly.
```

Comment:
I'll wait to the next iteration to go through each sentence, but here is another example where you can tighten up the language a lot.  Thy this: "The null coalescing operator (`??`), simplifies providing alternative values when an expression evaluates to `null`." (I also changed the use of "reference" because ?? works on nullable types as well.)